templates/elements/images/images.twig

```diff
@@ -0,0 +1 @@
+<img src="{{ src }}" alt="{{ alt }}">
```

Comment:
Just an example. The mixin can be used as needed, but this was mostly to provide sensible defaults and provide an example for the pattern library.

Reply:
Ok, I was going to suggest adding an `{{ attributes }}` variable so that developers could add additional attributes to the element tag.

eg: `<img src="{{ src }}" alt="{{ alt }}" {{ attributes }}>` 

But perhaps we don't to dismay future developers from using this template.

Reply:
Even more, do we even need an example for something like this?

Reply:
An example page may be needed in the future. Perhaps for now, if we are not intending the developer to use the .twig file we put the markup directly in to the sass comment.

Reply:
Cleaned up and moved.